torchtune/training/checkpointing/__init__.py

```diff
@@ -33,13 +41,21 @@
 ]
 
 __all__ = [
+    "get_largest_iter_folder",
     "FullModelHFCheckpointer",
     "FullModelMetaCheckpointer",
     "FullModelTorchTuneCheckpointer",
     "ModelType",
     "Checkpointer",
     "update_state_dict_for_classifier",
     "ADAPTER_CONFIG",
+    "ADAPTER_CONFIG_FNAME",
+    "ADAPTER_MODEL_FNAME",
+    "SHARD_FNAME",
+    "SAFETENSOR_INDEX_FNAME",
+    "SUFFIXES_TO_NOT_COPY",
+    "TORCH_INDEX_FNAME",
+    "REPO_ID_FNAME",
```

Comment:
I don't think these need to be exposed as public APIs right?

Reply:
agreed! I was following the pattern, but i guess that these other constants are used in multiple files. I will revisit this.